### PR TITLE
Sage Of Six Paths Naruto.MS

### PR DESCRIPTION
**Pull request by Supertre03**
Truly the most powerful form. So powerful it even allows Naruto to fly, or float. And in the Naruto Universe only the most powerful of beings are able to fly or float.

### CODE REVIEW

**Comment by malonnnn** [comment]
missing proof of powers

**Comment by Supertre03** [comment]
Proof Of Fast Speed and Jumping : https://www.youtube.com/watch?v=Eps7M86P3UY Go to 1:50 and watch from there

**Comment by Supertre03** [comment]
Proof of His Speed and Punch Damage His Punch Damage should be higher : https://www.youtube.com/watch?v=n_Z8TUatpMo

**Comment by Supertre03** [comment]
MORE PUNCH PROOF https://www.youtube.com/watch?v=FCH5HagBDvc

**Comment by Supertre03** [comment]
His Punch should Be Way HIGGHHERRR

File: CommandHelper/LocalPackages/commands/buffs/narutorikudosenninmode.ms [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@plr = player()
+
+
+
+@hero = associative_array(
+            'name': '§6Na§0r§6uto Ri§0ku§6do Se§0nn§6in M§0o§6de',
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/narutorikudosenninmode.ms]: This name does not match the name in permissions.yml

**Pull request merged**